src/app/(main)/[slug]/page.tsx

```diff
@@ -88,6 +92,13 @@ async function Page({ params }: Props) {
     );
   }
 
+  if (initialSalaryAndBenefitsPage.data) {
+    // TODO: add draft preview
```

Comment:
remember to also add preview 💡 

```
if(draftMode) {
return <SalaryAndBenefitsPreview initialData={initialSalaryAndBenefitsPage} />
}
```

Reply:
Assumed you wanted that in a separate PR, hence the TODO

Reply:
Created #492 now, can combine with this PR if you want

Reply:
My bad! Overlooked the todo!